<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrencyConverterMainWindow</class>
 <widget class="QMainWindow" name="CurrencyConverterMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="uservalue">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="currencyButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="useroutput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QComboBox" name="currencyComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="currencyComboBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
